src/client/app/effects/github.ts

```diff
@@ -134,12 +141,17 @@ export class GitHubEffects {
                 new Snippet.UpdateInfoAction({ id: snippetId, gist: gist.id, gistOwnerId: gist.owner.login })])
         )
         .catch(exception => {
+            let { gistOwnerId, id } = exception;
+            let action = (gistOwnerId && id)
+            ? new Snippet.UpdateInfoAction({id: id, gistOwnerId: gistOwnerId})
```

Comment:
Modified this to look like the multi-line statement in `scrubbedIfNeeded()` in `effects/snippet.ts`, but not sure if it's what you meant. Can change if necessary.